resources/assets/components/ReportbackUploader/index.js

```diff
@@ -74,6 +74,8 @@ class ReportbackUploader extends React.Component {
     this.setState({
       media: this.defaultMediaState,
     });
+
+    this.props.updateQuantityOrWhyState(reportback.impact ? reportback.impact : null, reportback.whyParticipated ? reportback.whyParticipated : null);
```

Comment:
Oh not really needed for this PR but this `ReportbackUploader` needs to be renamed to `PostUploader` and should be refactored to just use rogue terminology.

Reply:
ok! I made a card for that here: https://www.pivotaltracker.com/n/projects/2019429/stories/152350532